### PR TITLE
Create 2021-12-21-wt-open-research-fund.md

### PR DESCRIPTION
**Pull request by malvikasharan**
Draft for the Wellcome Trust grant announcement

### CODE REVIEW

File: _posts/2021-12-21-wt-open-research-fund.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- yochannah
+image: https://images.unsplash.com/photo-1533930086187-0fc58e5a92e2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987&q=80
+photos:
+  name: Photo by Elena Koycheva on Unsplash
+  url: https://unsplash.com/@lenneek
+---
+
+# Open Life Science is awarded Wellcome Trust Open Research Fund 2021
+
+We are delighted to share that Wellcome Trust has awarded the Open Research Fund to Open Life Science (OLS) to enable research and sustainability of the program for the next two years. Under the title **Systematic evaluation of community development through open science training and incentivising contextual mentorship in health research**, OLS was granted **GBP 99,000** to create a funded position for a programme coordinator and researcher, as well as pay for the overhead cost for the next four cohorts.
```
Comment (yochannah) [inline _posts/2021-12-21-wt-open-research-fund.md:17]:
```suggestion
We are delighted to share that Wellcome Trust has awarded the Open Research Fund to Open Life Science (OLS) to enable research and sustainability of the program for the next two years. Under the title **Systematic evaluation of community development through open science training and incentivising contextual mentorship in health research**, OLS was granted **GBP 99,999** to create a funded position for a programme coordinator and researcher, as well as pay for the overhead cost for the next four cohorts. 
```

Comment (yochannah) [reply _posts/2021-12-21-wt-open-research-fund.md:17]: I think?

File: _posts/2021-12-21-wt-open-research-fund.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+1. assessment of the long-term impact of OLS on the careers of the OLS participants and adoption of contextualised open science practices in research work in their regional contexts across low- and middle-income, as well as high-income countries, and
+1. publication of a series of reports capturing different outputs, building an open source evidence base and inviting support for the sustainability of OLS over the next 5 years.
+
+## Funded Position: Programme Coordinator and Researcher
+
+To maximise the impact of our project, we will recruit a staff member to join the OLS team as a Programme Coordinator and Researcher. They will also provide administrative support such as by helping ensure transparent communication with all the stakeholders, participating in outreach efforts, processing feedback from the cohorts and developing reports to share openly. Working with the OLS leadership team, they will also co-lead the impact studies and evaluation of OLS training and resources. A job announcement will be made in February 2022.
```
Comment (yochannah) [inline _posts/2021-12-21-wt-open-research-fund.md:31]:
```suggestion
To maximise the impact of our project, we will recruit a staff member to join the OLS team as a Programme Coordinator and Researcher. They will also provide administrative support such as by helping ensure transparent communication with all the stakeholders, participating in outreach efforts, processing feedback from the cohorts and developing reports to share openly. Working with the OLS leadership team, they will also co-lead the impact studies and evaluation of OLS training and resources. A job vacancy will be advertised in February 2022. 
```

to make it clear we're not announcing who was selected :)

File: _posts/2021-12-21-wt-open-research-fund.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Funded Position: Programme Coordinator and Researcher
+
+To maximise the impact of our project, we will recruit a staff member to join the OLS team as a Programme Coordinator and Researcher. They will also provide administrative support such as by helping ensure transparent communication with all the stakeholders, participating in outreach efforts, processing feedback from the cohorts and developing reports to share openly. Working with the OLS leadership team, they will also co-lead the impact studies and evaluation of OLS training and resources. A job announcement will be made in February 2022. 
+
+## Steering Committee
+
+Drawing from previous graduates, existing collaborators and volunteers in the OLS community, we will establish a steering committee. The committee will be engaged in co-creating a sustainability model for OLS beyond the lifetime of this funding as well as beyond the mentoring period of OLS projects (potentially in their local communities). The OLS team rely on continuous feedback from the OLS community and support to construct these pathways such that they are and will remain meaningful for them.
```
Comment (yochannah) [inline _posts/2021-12-21-wt-open-research-fund.md:35]: I've tried to reword 

```
 and support to construct these pathways such that they are and will remain meaningful for them.
```

But I don't really understand what it's trying to say, can we rephrase?

File: _posts/2021-12-21-wt-open-research-fund.md (hunk 2 above)
Comment (malvikasharan) [reply _posts/2021-12-21-wt-open-research-fund.md:31]: good point

File: _posts/2021-12-21-wt-open-research-fund.md (hunk 3 above)
Comment (malvikasharan) [reply _posts/2021-12-21-wt-open-research-fund.md:35]: it's taken as is from the application - I think Emmy wrote this bit.

File: _posts/2021-12-21-wt-open-research-fund.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Collaborating with Mission-Aligned Organisations
+
+We are looking to establish partnerships with mission-aligned organisations and projects interested in collaborating with OLS and adopting our training models in their institutions. This also includes funders seeking to support open science training for their funded projects or interested in supporting OLS cohort activities. Fruitful partnerships will enable the growth of OLS by helping explore new impact pathways to reach communities, particularly those currently underrepresented in open science.
```
Comment (yochannah) [inline _posts/2021-12-21-wt-open-research-fund.md:39]:
```suggestion
We are looking to establish partnerships with mission-aligned organisations and projects interested in collaborating with OLS and adopting our training models in their institutions. This includes funder and research institutes seeking to support open science training for their funded projects or interested in supporting OLS cohort activities. Fruitful partnerships will enable the growth of OLS by helping explore new impact pathways to reach communities, particularly those currently underrepresented in open science. To express early interest in a partnership, please email team@openlifesci.org.
```

File: _posts/2021-12-21-wt-open-research-fund.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Thank you! Connect with us.
+
+Thanks to all the OLS members who have supported the last four cohorts. We are very grateful for your support and engagement in the programme. We are proud of what we have done together and look forward to sharing this space for learning and growth in the coming years.
```
Comment (yochannah) [inline _posts/2021-12-21-wt-open-research-fund.md:43]:
```suggestion
Thanks to all the OLS members - mentors, participants, experts, and speakers - who have supported the last four cohorts. We are very grateful for your support and engagement in the programme. We are proud of what we have done together and look forward to sharing this space for learning and growth in the coming years.
```

File: _posts/2021-12-21-wt-open-research-fund.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+For any inquiries regarding the OLS program, upcoming job announcement and/or partnership, please email [team@openlifesci.org](mailto:team@openlifesci.org).
```
Comment (yochannah) [inline _posts/2021-12-21-wt-open-research-fund.md:45]:
```suggestion
For any inquiries regarding the OLS program, upcoming job vacancy and/or partnerships, please email [team@openlifesci.org](mailto:team@openlifesci.org).
```

File: _posts/2021-12-21-wt-open-research-fund.md (hunk 3 above)
Comment (malvikasharan) [reply _posts/2021-12-21-wt-open-research-fund.md:35]:
```suggestion
Drawing from previous graduates, existing collaborators and volunteers in the OLS community, we will establish a steering committee. The committee will be engaged in co-creating a sustainability model for OLS beyond the lifetime of this funding as well as beyond the mentoring period of OLS projects (potentially in their local communities). The OLS team will continue to seek feedback from the OLS community to improve pathways for engagement that are meaningful for all OLS participants.
```

Comment (yochannah) [reply _posts/2021-12-21-wt-open-research-fund.md:35]: I thought you might say that 😂

**Pull request merged**